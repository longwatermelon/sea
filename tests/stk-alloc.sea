fn main() -> int {
    let arr: int = stalloc(4,int);
    *arr = 10;
    *(arr+1*8) = 20;
    *(arr+2*8) = 30;
    *(arr+3*8) = 40;
    return *(arr+2*8);
}
