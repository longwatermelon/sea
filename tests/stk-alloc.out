RETURN_CODE:30
STDOUT:
